Odata-docs/client/bulk-update-operations.md

```diff
@@ -0,0 +1,245 @@
+---
+title: "Bulk Update Operations in OData Client"
+description: "How to perform bulk update operations in OData client"
+
+author: ElizabethOkerio
+ms.author: elokerio
+ms.date: 6/29/2023
+ms.topic: article
+ 
+---
+# Bulk Update Operations
+**Applies To**: [!INCLUDE[appliesto-odataclient](../includes/appliesto-odataclient-v7.md)]
+
+OData Client [v7.17.0](https://www.nuget.org/packages/Microsoft.OData.Client/) introduced support for bulk update operations.
+
+## What is a bulk update operation 
+
+A bulk update operation is an operation that allows you to perform several operations (GET, POST and PATCH) on resources of the same type and all their related resources using a single request. This can be highly efficient in situations where you need to update multiple resources and wish to avoid sending separate requests to process each resource. [Learn More](https://devblogs.microsoft.com/odata/bulk-operations-support-in-odata-web-api/).
+
+## Sample bulk update requests
+
+### Model
+To create the various sample requests in this tutorial, the following model classes will be used: 
+
+``` csharp
+public class Customer
+{
+    public int Id { get; set; }
+    public string Name { get; set; }
+    public int Age { get; set; }
+    public List<Order> Orders { get; set; }
+}
+
+public class Order
+{
+    public int Id { get; set; }
+    public decimal Amount { get; set; }
+}
+```
+### Sample Request 1
+This sample request demonstrates how to create a bulk update request in OData client to create 2 new customers in a single request. 
+
```

Comment:
In the statements above here, you mention about demonstrating "how to create a bulk update request"I think what should follow immediately after is the C# code snippet. The PATCH request and the JSON payload are the result of translating the C# statements. I think they should come after